parlai/core/script.py

```diff
@@ -203,10 +204,18 @@ def superscript_main(args=None):
     parser = _SupercommandParser(
         False, False, formatter_class=_SuperscriptHelpFormatter
     )
+    helpall_help_str = 'List all commands, including advanced ones.'
     parser.add_argument(
         '--helpall',
         action='helpall',
-        help='show all commands, including advanced ones.',
```

Comment:
nit: I prefer the inline strings, if you don't mind.

Reply:
ok, I made the change as string was repeated in L225 but slightly different text.

Reply:
i'll change it back

Reply:
ok 99be94f85b makes it an inline string, but still edits to make the help string and description consistent

Reply:
Great, thanks for the PR mate!